.gitignore

```diff
@@ -13,3 +13,6 @@ env/
 
 # rope
 .ropeproject/
+
+# coverage
+.coverage
```

Comment:
This is an unrelated change, but sure I'll merge it..

Reply:
Tbh, I'm new to 'coverage' as a tool. When it ran, it created a .coverage file that git wanted to add and commit, so I assumed it had to be ignored?

Reply:
Yes, it can be ignored. But it has nothing to do with Python 3.